x-pack/plugins/observability_solution/logs_explorer/public/customizations/custom_field_list.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataViewField } from '@kbn/data-views-plugin/common';
+import * as constants from '../../common/constants';
+
+export const smartFields = [
+  {
+    name: constants.RESOURCE_FIELD,
+    displayName: constants.RESOURCE_FIELD,
+    type: 'smart_field',
+  } as DataViewField,
+  {
+    name: constants.CONTENT_FIELD,
+    displayName: constants.CONTENT_FIELD,
+    type: 'smart_field',
+  } as DataViewField,
+];
```

Comment:
It would be better to use `new DataViewField(...)` to ensure the type and structure